samples/python/visual_language_chat/video_to_text_chat.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import argparse
+import numpy as np
+import cv2
+import openvino_genai
+from openvino import Tensor
+from pathlib import Path
+
+
+def streamer(subword: str) -> bool:
+    '''
+
+    Args:
+        subword: sub-word of the generated text.
+
+    Returns: Return flag corresponds whether generation should be stopped.
+
+    '''
+    print(subword, end='', flush=True)
+
+    # No value is returned as in this example we don't want to stop the generation in this method.
+    # "return None" will be treated the same as "return openvino_genai.StreamingStatus.RUNNING".
+
+
+def read_video(path: str, num_frames: int = 10) -> Tensor:
+    '''
+
+    Args:
+        path: The path to the video.
+
+    Returns: the ov.Tensor containing the video.
+
+    '''
+    cap = cv2.VideoCapture(path)
+
+    frames = []
+
+    while cap.isOpened():
+        ret, frame = cap.read()
+        if not ret:
+            break
+
+        frames.append(np.array(frame))
+    cap.release()
+
+    indices = np.arange(0, len(frames), len(frames) / num_frames).astype(int)
+    frames = [frames[i] for i in indices]
```

Comment:
What this reindexing is for?

Reply:
It is needed to return a uniform subset of frames instead of all frames from video. Llava-next-video was trained on frames subsets of size 32  and will return trash if full video is passed, that has more frames than that number. In the model card 10 frames are used as an example, so I used 10 frames here as a default number.

Reply:
Done.